src/instance.ts

```diff
@@ -526,18 +693,20 @@ class Instance extends common.ServiceObject {
         gaxOpts: query,
       },
       // tslint:disable-next-line only-arrow-functions
-      function(err, databases) {
+      function(err: Error, databases: IDatabase[]) {
         if (databases) {
           arguments[1] = databases.map(database => {
-            const databaseInstance = self.database(database.name);
+            const databaseInstance = self.database(database.name!);
             databaseInstance.metadata = database;
             return databaseInstance;
           });
         }
-        callback.apply(null, arguments);
+        // tslint:disable-next-line no-any
```

Comment:
Shouldn't need this, please refer to the PubSub [example](https://github.com/googleapis/nodejs-pubsub/blob/master/src/pubsub.ts#L631) on how we've handled paginated methods like this.